src/test/java/com/eleks/academy/whoami/service/GameServiceImplTest.java

```diff
@@ -229,11 +229,14 @@ void startGameTest() {
 		var expectedGame = GameDetails.builder()
 				.id(id)
 				.status(GameStatus.IN_PROGRESS)
-				.currentTurn("player1")
+				.currentTurn(List.of())
```

Comment:
list of what?

Reply:
Resolved
I just forgot to update this part of the test after updating TurnImpl constructor